### PR TITLE
The specfiles are set as a comma-separated list

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Signed-off-by: Frantisek Lachman <flachman@redhat.com>

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
pre-commit.ci run

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      - id: black
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.7.1
+    hooks:
+      - id: prettier
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: check-added-large-files
+        exclude: tests_recording/test_data/test_api/
```
Comment (jpopelka) [inline .pre-commit-config.yaml:22]:
```suggestion
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        additional_dependencies:
+          [
+            types-click,
+            types-pkg_resources,
+            types-requests,
+            types-tabulate,
+            types-Deprecated,
+            types-PyYAML,
+            types-cachetools,
+          ]
```
Comment (jpopelka) [inline .pre-commit-config.yaml:55]:
```suggestion
```

File: .pre-commit-config.yaml (hunk 1 above)
Comment (lachmanfrantisek) [reply .pre-commit-config.yaml:22]: Ah, I missed this occurrence...

**Review by nforro — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+      - id: trailing-whitespace
+  - repo: https://github.com/PyCQA/flake8
+    rev: 5.0.4
+    hooks:
+      - id: flake8
+        args:
+          - --max-line-length=100
+          - --per-file-ignores=files/packit.wsgi:F401,E402
```
Comment (jpopelka) [inline .pre-commit-config.yaml:39]: actually, this line can go as well, so next time :)

Comment (lachmanfrantisek) [reply .pre-commit-config.yaml:39]: Here it is: https://github.com/packit/prepare-release/pull/2

**Pull request merged**